proxmox/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ec8be79a-7f8a-4d9b-b238-907247dfe346",
+  "app_id": "proxmox",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor Proxmox with Datadog",
+    "title": "Proxmox",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Category::Provisioning",
+      "Offering::Integration",
+      "Submitted Data Type::Metrics"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": true,
+      "source_type_id": 33153735,
+      "source_type_name": "Proxmox",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "proxmox.",
+        "check": "",
```

Comment:
Is there a metric that is always submitted and represents that the integration is working? If so, please specify that here